packages/codegen-ui-react/lib/forms/form-renderer-helper/value-props.ts

```diff
@@ -74,8 +74,12 @@ export const renderDefaultValueAttribute = (
     expression = factory.createJsxExpression(undefined, convertedValueAttributeMap[dataType](identifier));
   }
 
+  const componentsWithValue: string[] = ['TextField', 'TextAreaField'];
```

Comment:
For things like this, can we use `Set` instead of `Array`. Set.has() is O(1) vs O(n) of an array. This is a really small list but we can start getting into the habit especially if codegen runs as a service at some point, performance is going to matter more.

Reply:
Makes sense. Will do in a follow-up